src/AppBundle/Controller/TodoController.php

```diff
@@ -0,0 +1,242 @@
+<?php
+/*
+ * Routes for accessing the To do info.
+ *
+ * All of these paths are behind the firewall for ROLE_USER.
+ *
+ * All of these paths consume and send application/json
+ */
+
+namespace AppBundle\Controller;
+
+use AppBundle\ControllerTemplates\JSONController;
+use AppBundle\ControllerTemplates\JSONControllerException;
+use AppBundle\Document\Todo;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class TodoController extends JSONController
+{
+    /**
+     * @Route("/todo", name="get_todo")
+     * @Method({"GET"})
+     *
+     * Gets the ids of all items.
+     *
+     * @return JsonResponse
+     */
+    public function getTodoAction()
+    {
+        //Get all to do for this user
+        $todos = $this->dm()->createQueryBuilder('AppBundle:Todo')
+            ->field('userId')->equals($this->getUser()->getIdString())
+            ->getQuery()->execute();
+
+        //get the _id, title, and completed
+
+        $output = array();
+        foreach ($todos as $todo) {
+            $temp = array();
+            $temp['_id'] = '' . $todo->getId();
+            $temp['completed'] = $todo->getCompleted();
+            $temp['title'] = $todo->getTitle();
+            array_push($output, $temp);
+        }
```

Comment:
Pat hates the old array syntax, try to use []'s whenever possible. Additionally, you could have done:
$output[] = [
   '_id' => $todo->getId(),
  'completed' => $todo->getCompleted(),
  'title' => $todo->getTitle()
]
instead of the array_push 
but no biggy